### PR TITLE
Ready for Heroku Deploy

### PR DESCRIPTION
**Pull request by bryantee**
:rocket: :rocket: :rocket:

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -59,11 +63,15 @@ app.delete('/items/:id', function(req, res) {
...
 app.put('/items/:id', jsonParser, function(req, res) {
   var id = parseInt(req.params.id);
   var body = req.body;
-  var itemName; // the new name
+  var itemName;
   var idFound = false;
+  if (id != req.body.id) {
+    return res.sendStatus(400);
+  }
   if (req.body.hasOwnProperty("name")) {
```
Comment (strothj) [inline server.js]: If a `put` request is sent without a `name` field, I think this handler will not send back a response. Might need to do a check for the existence of the `name` field so a `400` response code can be sent back.

**Review by strothj — APPROVED** [review]
Looks good.

File: server.js (hunk 1 above)
Comment (bryantee) [reply server.js]: @strothj Ahhh, good call. On it.
#3 issue added

Comment (bryantee) [reply server.js]: Closed. Thanks @strothj

**Pull request merged**